client/src/views/Dashboard/Summary/index.js

```diff
@@ -423,6 +438,204 @@ export default class Summary extends PureComponent {
+                label: 'Child living with step parents divorced parents single parent',
+                value: data.totalChildLivingWithStepParentsDivorcedParentsSingleParent,
+            },
+            {
+                type: 'Extreme Deprivation',
+                label: 'Extreme Deprivation Total',
+                value: data.totalExtremeDeprivation,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Disaster Prone Area',
+                value: data.totalChildLivingInDisasterProneArea,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Areas With Conflict',
+                value: data.totalChildLivingInAreasWithConflict,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Living in Former War Zones',
+                value: data.totalChildLivingInFormerWarZones,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Child Affected by Epidemic',
+                value: data.totalChildAffectedByEpidemic,
+            },
+            {
+                type: 'Catastrophe/Disaster',
+                label: 'Catastrophe/  Disaster Total',
+                value: data.totalCatastropheDisaster,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child with Disability',
+                value: data.totalChildWithDisability,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Belongs to a Marginalized Group',
+                value: data.totalChildBelongsToA_marginalizedGroup,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Refugee Children of Refugees Migrant',
+                value: data.totalChildRefugeeChildrenOfRefugeesMigrant,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Delinquent',
+                value: data.totalChildDelinquent,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Whose Parents are Imprisoned',
+                value: data.totalChildWhoseParentsAreImprisoned,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Without Birth Registration',
+                value: data.totalChildWithoutBirthRegistration,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Child Living in Isolation',
+                value: data.totalChildLivingInIsolation,
+            },
+            {
+                type: 'Serious Discrimination',
+                label: 'Total Serious Discrimination',
+                value: data.totalSeriousDiscimination,
+            },
+        ].filter(v => v.value);
```

Comment:
It's better to filter numbers using `v => isDefined(v.value)` as `0` is a falsy value

Reply:
yeah 0 is also filtered